src/dune_lang/dune_lang.mli

```diff
@@ -146,9 +148,11 @@ module Cst : sig
 
   val concrete : Ast.t -> t
 
+  val to_sexp : t -> sexp option
```

Comment:
I know this is just following the convention in this module, but `sexp` is actually the wrong name. `Sexp.t` is just a debugging format that we use. Here it refers to dune's sexp based language which is actually `Dune_lang.t`.

Earlier, the distinction between the two wasn't so clear so things were messy. We have to think of a new name here and for other misuses of `sexp`.

Reply:
Maybe something along the lines of `pure_ast`?

Reply:
Unfortunately, the `ast` is also taken. It's just the dune language + location information.

Reply:
Yes, that's why I suggested the `pure` prefix.